package.json

```diff
@@ -73,5 +90,17 @@
   "optionalDependencies": {
     "bufferutil": "^4.0.3",
     "utf-8-validate": "^5.0.4"
+  },
+  "build": {
+    "appId": "com.awala.gateway",
```

Comment:
Can you please use `tech.relaycorp.gateway` here instead? I like to avoid using "awala"/"relaynet" in the namespace to imply that this just our implementation of the protocol suite but others can/will create their own.

Reply:
And this is done.